packages/web/src/scss/components/Offcanvas/README.md

```diff
@@ -0,0 +1,79 @@
+# Offcanvas
+
+Offcanvas is a sidebar component that can be toggled via JavaScript to appear from the left, right, or bottom edge of the viewport.
+Buttons or anchors are used as triggers that are attached to specific elements you toggle, and data attributes are used to invoke JavaScript plugin.
+
+## JavaScript Plugin
+
+For full functionality, you need to provide Spirit JavaScript, which will handle
+toggling of the Header component:
+
+```html
+<script src="node_modules/@lmc-eu/spirit-web/js/cjs/spirit-web.min.js" async></script>
+```
+
+## Usage
+
+### Via data attributes
+
+Add data-spirit-toggle="offcanvas" and a data-spirit-target or href to the element to automatically assign control of one Offcanvas element.
+The data-spirit-target attribute accepts a CSS selector to apply the Offcanvas to.
+
+### Via JavaScript
+
+Enable manually with:
+
+```js
+const offcanvasElementList = [].slice.call(document.querySelectorAll('.offcanvas'));
+const offcanvasList = offcanvasElementList.map(function (offcanvasElement) {
+  return new Offcanvas(offcanvasElement);
+});
+```
+
+### Options
+
+Options can be passed via data attributes or JavaScript.
+For data attributes, append the option name to data-spirit-, as in data-spirit-breakpointDesktop="".
+
+| Name              | Type   | Default | Description                                                       |
+| ----------------- | ------ | ------- | ----------------------------------------------------------------- |
+| breakpointDesktop | string | 1280 \* | Desktop breakpoint on which opened Offcanvas automatically closes |
+
+\*: Default value of the design token `others.breakpoints.desktop`. Can be also changed via `--spirit-breakpoint-desktop` CSS custom property.
+
+### Methods
+
+Activates your content as an offcanvas element. Accepts an optional options object.
+
+You can create an offcanvas instance with the constructor, for example:
+
+```js
+const customOffcanvas = document.getElementById('customOffcanvas');
+const spiritOffcanvas = new Offcanvas(customOffcanvas);
+```
+
+| Method              | Description                                                                                                                                                                                              |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| toggle              | Toggles an Offcanvas element to shown or hidden. Returns to the caller before the Offcanvas element has actually been shown or hidden (i.e. before the show.offcanvas or hidden.offcanvas event occurs). |
```

Comment:
Place add backticks to all items in Method column and in the Event type too.

Reply:
Added.